Dockerfile

```diff
@@ -1,73 +1,32 @@
-FROM centos:centos6
-# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
-RUN groupadd -r postgres && useradd -r -g postgres postgres
+FROM quay.io/centos/centos:stream8 as bingo
+
+RUN \
+  dnf update -y && \
+  dnf upgrade -y && \
+  dnf install -y tar git cmake gcc-c++ && \
+  dnf module enable -y postgresql:13 && \
+  dnf install postgresql-server-devel postgresql-devel -y && \
+  dnf config-manager --set-enabled powertools && \
+  dnf install -y libstdc++-static && \
+  dnf clean all
+
+ENV INDIGO_VERSION master
```

Comment:
Bit of a misnomer to set the version to the master branch rather than a particular tag / release, but I'm alright with the default being latest.